ro_help/mobilpay/admin.py

```diff
@@ -60,4 +60,7 @@ def get_queryset(self, request):
         return qs
 
     def order_id(self, obj):
+    try:
```

Comment:
Misaligned here.
BTW, you can use `PaymentOrder.DoesNotExist`, in order to be more specific.